src/client/pages/Geo/GeoMap/GeoMapMenuData/MapVisualizerPanel/ProtectedAreaLayerSection/CustomAssetControl/CustomAssetControl.scss

```diff
@@ -0,0 +1,137 @@
+@import 'src/client/style/partials';
+@import 'src/client/styles/spinner';
+
+.custom-asset-data-visualizer-agreement-levels-control {
+  display: flex;
+  flex-direction: column;
+  padding: 5px 25px 30px 30px;
+}
+
+.custom-asset-data-visualizer-agreement-levels-boxes {
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+  justify-content: space-between;
+
+  .custom-asset-data-visualizer-agreement-level {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 10px;
+
+    &.disabled {
+      color: #bdbdbd;
+    }
+  }
+
+  .custom-asset-data-visualizer-agreement-levels-box {
+    appearance: none;
+    width: 30px;
+    height: 30px;
+    display: grid;
+    place-content: center;
+    background-color: #e7e7e7;
+    border: 1px solid #949495;
+    border-radius: 3px;
+    cursor: pointer;
+
+    &::before {
+      content: '';
+      width: 1em;
+      height: 1em;
+    }
+
+    &:checked {
+      background-color: #262627;
+      border-color: #5f5f5f;
+      box-shadow: none;
+    }
+
+    &:disabled {
+      background-color: #f8f7f7;
+      border-color: #dedfdf;
+      box-shadow: 0 0 10px 2px #efeeee inset;
+      cursor: default;
+    }
+  }
+
```

Comment:
Any chance to use the global color variables ?

Reply:
Yes, fixed this and a bug in the new commit.